recipes/rooki/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "rooki" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d9d05fc81046ad1ed5ee9528728644527e812e6822fdd11ce6aeee8426341afe
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.9
+    - birdy >=0.8.2
+    - beautifulsoup4
+    - pymetalink
+    - owslib
+    - pyproj >=3.0.1
+    - netcdf4
+    - xarray >=0.16
+    - cf_xarray
```

Package metadata suggests:

```suggestion
    - python >=3.9
    - click >=6.0
    - birdhouse-birdy >=0.8.2,<0.9.0
    - netcdf4 >=1.5
    - xarray >=0.16
    - cf_xarray
    - beautifulsoup4 >=4.9.1
    - pymetalink >=6.2
```

Can you check upstream @cehbrecht?